Assets/Scripts/Models/Furniture.cs

```diff
@@ -384,7 +384,10 @@ public bool HasPower()
         {
             if (World.current.powerSystem.RequestPower(this) == true)
             {
-                return true;
+                if (World.current.powerSystem.PowerLevel > 0)
```

Comment:
`return World.current.powerSystem.Powerlevel > 0;`

No need for the `if`-`else`